sw/host/hsmtool/src/util/wrap.rs

```diff
@@ -42,14 +65,20 @@ impl Wrap {
                 PkcsMgfType::MGF1_SHA256,
                 PkcsOaepSource::empty(),
             ))),
+            Wrap::VendorThalesAesKwp => {
+                Ok(Mechanism::VendorDefined(VendorDefinedMechanism::new::<()>(
+                    MechanismType::new_vendor_defined(CKM_VENDOR_DEFINED | 0x00000171)?,
```

Comment:
Should we create a symbolic constant for this magic number?  (maybe not? It's kindof obvious from the context that this is the ThalesAesKwp value)

Reply:
SGTM. Added constant.